src/ui/pages/WalletSelect/WalletSelect.tsx

```diff
@@ -88,6 +144,9 @@ export function WalletSelect() {
       <PageColumn>
         {title}
         <Spacer height={10} />
+        {walletGroups.length > 1 ? (
```

Comment:
We need to check here that there is more then 1 not readonly address to show this line. May be it can be done inside the component